src/scss/_spark-theme.scss

```diff
@@ -51,20 +51,31 @@ body {
 
 .docs-c-SkipNav {
   position: absolute;
-  top: 0;
+  top: -150px;
```

Comment:
Looks like you changed some sass for the skip nav but I didn't see that in the issue. Whats this change for?

Reply:
made the PR in the wrong branch, this is not part of the task